pkgs/development/tools/gotools/default.nix

```diff
@@ -1,60 +1,40 @@
-{ lib, buildGoModule, fetchgit }:
+{ lib, buildGoModule, fetchFromGitHub }:
 
 buildGoModule rec {
   pname = "gotools";
-  version = "0.1.10";
+  version = "0.7.0";
 
-  src = fetchgit {
+  # using GitHub instead of https://go.googlesource.com/tools because Gitiles UI is to basic to browse
```

Comment:
What do you mean by "UI is to basic to browse"? I feel there's a typo here.

Reply:
Gitiles can only show the bare minimum information about a git repo and things like commit diffs are behind extra clicks. I just wanted to avoid that people switch back again.